Assignment.Interfaces/Assignment.Interfaces.csproj

```
@@ -3,8 +3,29 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>

```

```suggestion
    <TargetFramework>netstandard2.1</TargetFramework>
```
This will fix your version error